maintainers/maintainer-list.nix

```diff
@@ -4755,6 +4755,12 @@
     githubId = 40566146;
     name = "Jonas Braun";
   };
+  j-hui = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
Thanks for the review! Should I force push the new commits?

Reply:
yes, please

Reply:
Done! I was unable to sort it alphabetically globally, because the maintainer list seems to already be out of order: `j-brn`, `j-keck`, and `j03` are positioned between `jasoncarr` and `jackgerritts`, for some reason.